internal/terraform/node_resource_plan_instance.go

```diff
@@ -702,7 +702,7 @@ func (n *NodePlannableResourceInstance) importState(ctx EvalContext, addr addrs.
 		}
 
 		// We skip the read and further validation since we make up the state
-		// of the imported resource anyways.
+		// of the imported resource anyways.
```

Comment:
There was a weird white space-like character here that my IDE highlighted for removal